platform/ios/src/MLNMapView.h

```diff
@@ -1364,6 +1364,18 @@ MLN_EXPORT
  */
 - (CGPoint)anchorPointForGesture:(UIGestureRecognizer *)gesture;
 
+
+/**
+ * Sets a LatLngBounds that constraints map transformations to this bounds.
+ * @param latLngBounds the bounds to constrain the map with
+ */
+- (void)setLatLngBounds:(MLNCoordinateBounds)latLngBounds;
+
+/**
+ * Clears the bounds that were set via `setLatLngBounds`
+ */
+- (void)clearLatLnBounds;
```

Comment:
Hi, thanks for your work! I'd like to mention that there may be a typo in the `clearLatLnBounds` API. Should it be `clearLatLngBounds`? The changelog and GitHub release notes have the same typo.